crates/wasmcloud-host/src/host.rs

```diff
@@ -232,6 +234,7 @@ impl Host {
             rpc_timeout: self.rpc_timeout,
         };
         mb.send(init).await?;
+        eprintln!("Host sent init...");
```

Comment:
This could read better for debugging as something like:
```
Host sent messagebus initialization message ...
```

Reply:
removed